Cannot find module 'lusca'

When I try to run 'node server.js' I get the following error:

```
module.js:340
    throw err;
    ^
Error: Cannot find module 'lusca'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/brasilikum/express4-bootstrap-starter/app/config/express.js:11:24)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

--- a/package.json
+++ b/package.json
@@ -67,7 +67,8 @@
     "cheerio": "^0.17.0",
     "email-templates": "^1.1.1",
     "nodemailer": "^1.2.0",
-    "ejs": "^1.0.0"
+    "ejs": "^1.0.0",
+    "lusca": "^1.0.1"
   },
   "devDependencies": {
     "mocha": "^1.21.4",

